patroni/ctl.py

```diff
@@ -646,3 +646,59 @@ def configure(config_file, dcs, namespace):
     config['dcs_api'] = str(dcs)
     config['namespace'] = str(namespace)
     store_config(config, config_file)
+
+
+def touch_member(config, dcs):
+    ''' Rip-off of the ha.touch_member without inter-class dependencies '''
+    p = Postgresql(config['postgresql'])
+    p.set_state('running')
+    p.set_role('master')
+
+    api = RestApiServer(None, config['restapi'])
+    data = {
+        'conn_url': p.connection_string,
+        'api_url': api.connection_string,
+        'state': p.state,
+        'role': p.role
+    }
+    try:
+        dcs.touch_member(json.dumps(data, separators=(',', ':')), permanent=True)
+    except DCSError:
```

Comment:
`touch_member` if not raising exceptions... 
:returns: `!True` on success otherwise `!False`